contracts/MerkleLib.sol

```diff
@@ -64,6 +66,7 @@ library MerkleLib {
         uint256 chunkIdx
     ) internal pure returns (bytes32[] memory) {
         uint256 nChunks = 1 << nChunkBits;
+        require(chunkIdx < nChunks, "index must be within the scope of number of chunks");
```

Comment:
We should reduce the length of the error message to save gas cost.

Reply:
ACK. will shorten it.